datasets/dataforgood-fb-hrsl.yaml

```diff
@@ -0,0 +1,27 @@
+Name: High Resolution Population Density Maps + Demographic Estimates by CIESIN and Facebook
+Description: |
+  The population of the world, allocated to 1 arcsecond blocks. This
+  refines CIESIN’s Gridded Population of the World project, using
+  machine learning models on high-resolution worldwide Digital Globe
+  satellite imagery.
+Documentation: https://dataforgood.fb.com/tools/population-density-maps/
+Contact: disastermaps@fb.com
+UpdateFrequency: Quarterly
+Tags:
+  - population
+  - demographics
+  - machine learning
+  - aerial imagery
+  - satellite imagery
+  - image processing
+  - geospatial
+  - CIESIN
+  - Facebook
+  - DigitalGlobe
+  - Columbia
+License: http://www.gdeltproject.org/about.html#termsofuse
```

Comment:
Just confirming this is the license you'd like to use?

Reply:
Thanks, I think I left this unmodified from a copy-paste. Updated with info from https://www.ciesin.columbia.edu/data/hrsl/